taskvine/src/manager/vine_manager.c

```diff
@@ -3674,6 +3674,7 @@ struct vine_manager *vine_ssl_create(int port, const char *key, const char *cert
 	q->minimum_transfer_timeout = 60;
 	q->transfer_outlier_factor = 10;
 	q->default_transfer_rate = 1 * MEGABYTE;
+	q->transfer_retry_interval = 5;
```

Comment:
`q->last_failure_retry_interval = DEFAULT_LAST_FAILURE_RETRY_INTERVAL;`  and `#define` accordingly.
We can use the same value for the task retries, etc.